src/parser/csql_lexer.l

```diff
@@ -757,6 +757,9 @@ int yybuffer_pos;
 [rR][eE][uU][sS][eE]_[oO][iI][dD]					{ begin_token(yytext);
 										csql_yylval.cptr = pt_makename(yytext);
 										return REUSE_OID; }
+[dD][oO][nN][tT]_[Rr][Ee][uU][sS][eE]_[oO][iI][dD]						{ begin_token(yytext);
```

Comment:
In the case of other items, the braces for begin_token usually start at column 73, so it would be good to match the same.